R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -246,8 +246,8 @@ test_that("read/write csv as DataFrame", {
   mockLinesCsv <- c("year,make,model,comment,blank",
                    "\"2012\",\"Tesla\",\"S\",\"No comment\",",
                    "1997,Ford,E350,\"Go get one now they are going fast\",",
-                   "2015,Chevy,Volt",
-                   "NA,Dummy,Placeholder")
+                   "2015,Chevy,Volt,,",
```

Comment:
yea, so I think i'ld be better to take the longer cases in this pr, and then we need to discuss more about the shorter cases in another pr.